src/banded_global_aligner.cpp

```diff
@@ -0,0 +1,1698 @@
+//
+//  banded_global_aligner.cpp
+//  
+//
+//  Created by Jordan Eizenga on 7/25/16.
```

Comment:
@adamnovak I think the objective is to enable a faster alignment mode when we have a good idea of the overall structure of the alignments.

I guess you mean we should add commentary into the source. That would be helpful.